### ISSUE TITLE
Syntax error: Extra closing curly brace in 'inputStyleNeutral' CSS component

### ISSUE THREAD

**Issue by annaghi** [WordPress/gutenberg#59529]
### Description

The CSS `border` property incorrectly includes an extra closing curly brace.

Incorrect syntax:
`border: ${ CONFIG.borderWidth } solid ${ COLORS.ui.border } };`

Correct syntax:
`border: ${ CONFIG.borderWidth } solid ${ COLORS.ui.border };`

### Step-by-step reproduction instructions

Navigate to `packages/components/src/utils/input/base.js` and locate the `inputStyleNeutral` component:

```js
export const inputStyleNeutral = css`
	box-shadow: 0 0 0 transparent;
	transition: box-shadow 0.1s linear;
	border-radius: ${ CONFIG.radiusBlockUi };
	border: ${ CONFIG.borderWidth } solid ${ COLORS.ui.border } };    <----- extra '}' at the end
	${ reduceMotion( 'transition' ) }
`;
```

[Visit https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/utils/input/base.js#L17](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/utils/input/base.js#L17)

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by ShubhamPatilsd** [comment]
Our students (beginners) at https://labs.codeday.org will be taking a go at this in the coming week :)

**Comment by SahilThakur02** [comment]
@annaghi can I start working on it?

**Comment by annaghi** [comment]
@ShubhamPatilsd Have you (or your students) started to work on this issue? @SahilThakur02 is also offering help.

**Comment by oceantober** [comment]
My team and I from Code Day are working on this issue! We will be starting this week.

**Comment by mirka** [comment]
Closed by #59744
